esp/src/eclwatch/WUDetailsWidget.js

```diff
@@ -559,7 +559,7 @@ define([
         refreshActionState: function () {
             var isArchived = this.wu.get("Archived");
             this.setDisabled(this.id + "AutoRefresh", isArchived || this.wu.isComplete(), "iconAutoRefresh", "iconAutoRefreshDisabled");
-            registry.byId(this.id + "Save").set("disabled", isArchived || !this.wu.isComplete() || this.wu.isDeleted());
+            registry.byId(this.id + "Save").set("disabled", isArchived || !this.wu.isComplete() && !this.wu.isBlocked() || this.wu.isDeleted());
```

Comment:
This line is fine - but I had to double check the operator precedence to be sure (https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Operators/Operator_Precedence) - the following would be easier to read (for me):
```
registry.byId(this.id + "Save").set("disabled", isArchived || (!this.wu.isComplete() && !this.wu.isBlocked()) || this.wu.isDeleted());
```